frontend/js/src/playlists/components/ImportPlaylistModal.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from "react";
+import NiceModal, { useModal } from "@ebay/nice-modal-react";
+import { toast } from "react-toastify";
+
+import GlobalAppContext from "../../utils/GlobalAppContext";
+import { ToastMsg } from "../../notifications/Notifications";
+
+type ImportPlaylistModalProps = {
+  playlist?: JSPFPlaylist;
+  initialTracks?: JSPFTrack[];
```

Comment:
This is just copy pasted from another modal, both of these props are not needed.
Will also need to be removed from line 21.